view/rooms/app.js

```diff
@@ -300,12 +300,15 @@ function createFooter(bot) {
   const footer = document.createElement('h3');
   const linkedElement = document.createElement('a');
   const gitHubImage = document.createElement('img');
+  const botVersion = document.createElement('span');
 
   footer.className =
     'slds-section__title ' +
     'slds-p-horizontal_small ' +
     'slds-theme_shade ';
 
+  botVersion.innerHTML = 'Version '+bot.version;
```

Comment:
spaces around plus sign

Reply:
jscs does not ENFORCE style for everything under /view but you should run it anyway and bring your code up to style standards